src/js/themes/hpe.js

```diff
@@ -1938,29 +1959,33 @@ const buildTheme = (tokens, flags) => {
       wide: {
         width: {
           min: '336px', // 336 + 24 (margin) + 24 (margin) = 384 (e.g. 'medium')
-          max: 'xxlarge',
```

Comment:
It looks like 'xlarge' re-maps to '3xlarge'. Does 'xxlarge' also re-map to '3xlarge'?

Reply:
Whoops the mapping table was off for that. I just fixed it. But this remapping xxlarge --> 3xlarge is correct.